_platforms/arm.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
I'm not entirely happy calling this arm when we have both specific dialects as well as platform filesystems.  I'd almost rather split this section into two and have one that just talks about that you can get arbitrary rootfs tarballs, and another page that talks about SBCs and links to the relevant pages in the documentation.

This would also be nice as I'd like to add a tab for the various cloud platforms that Void supports.

Reply:
I have split ARM into 2 sections and sketched content for both, would appreciate a review on those.

re cloud images: I only see GCP tarball at https://alpha.de.repo.voidlinux.org/live/current/ , what other platforms we want to show on the downloads  page?

Reply:
Both GCP and DigitalOcean are known to work and are some semblance of supported (Void itself has infrastructure running on DigitalOcean).  For both though the images are not distributed by default, so just having a place to link back to the docs on how to generate the image on demand would be great.